### PR TITLE
fix(window): avoid `global.window` redefinition

### LINKED ISSUE #89: Test fails on Node.js 18
The `npm test` fails on Node.js 18.

BTW, versions in `.travis.yml` need to be updated.

### How to reproduce

```shell
cd jest-canvas-mock
nvm use 18
npm test
```

This error message below will show up from all tests:

```shell
TypeError: Cannot redefine property: window
```

### PR DESCRIPTION
**Pull request by LitoMore**
### Changes

- Fix #89, also closes #84
- Move to GitHub Actions (related to #90)

### CODE REVIEW

**Review by langpavel — APPROVED** [review]
Great work!

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: CI
```
Comment (hustcc) [inline .github/workflows/main.yml:1]: use name `build`

File: src/window.js [hunk 2]
```diff
@@ -56,6 +56,4 @@ export default (win) => {
   if (!win.createImageBitmap) win.createImageBitmap = createImageBitmap;
 
   mockPrototype();
-
-  return win;
```
Comment (hustcc) [inline src/window.js:60]: keep this line.

**Comment by hustcc** [comment]
add a worker mock for the test case. @LitoMore

File: src/window.js (hunk 2 above)
Comment (LitoMore) [reply src/window.js:60]: There is no reason to keep this. You didn't use return value, right?

Comment (hustcc) [reply src/window.js:60]: 1. the function get the `window`, and return a new window. But this method is special, it is mutable, return the same `window`
2. break change for strict mode

Comment (LitoMore) [reply src/window.js:60]: I can't fully agree with you but updated as you wish.

This is not my main issue to discuss, I just want to fix this bug for Node.js 18 ASAP.

**Comment by hustcc** [comment]
v2.4.0 released.

**Pull request merged**